pkgs/servers/web-apps/plausible/default.nix

```diff
@@ -0,0 +1,88 @@
+{ lib, stdenv, beamPackages, fetchFromGitHub, glibcLocales, cacert
+, mkYarnModules, nodejs, fetchpatch, nixosTests
+}:
+
+let
+  pname = "plausible";
+  version = "1.3.0";
+  name = "${pname}-${version}";
+
+  src = fetchFromGitHub {
+    owner = "plausible";
+    repo = "analytics";
+    rev = "v${version}";
+    sha256 = "03lm1f29gwwixnhgjish5bhi3m73qyp71ns2sczdnwnbhrw61zps";
+  };
+
+  # TODO consider using `mix2nix` as soon as it supports git dependencies.
+  mixFodDeps = beamPackages.fetchMixDeps {
+    pname = "${pname}-deps";
+    inherit src version;
+    sha256 = "18h3hs69nw06msvs3nnymf6p94qd3x1f4d2zawqriy9fr5fz7zx6";
+    patches = [ ./ecto_sql-fix.patch ];
+  };
+
+  yarnDeps = mkYarnModules {
```

Comment:
Just curious here. Why did you choos yarn over node2nix for example?
I thought that the project was using yarn, but it's actually using npm by default.
If it's too much work, feel free to leave it as is. I'm thinking we should perhaps just add a comment as to why this choice was made in case future maintainers want to change this.

Reply:
From my experience `yarn2nix` is wayy more robust than `node2nix` so `yarn2nix` is usually my first choice :)

Reply:
Maybe just add a comment saying that here yarn2nix was chosen over node2nix from the maintainers experience regarding stability.
The idea would be to make it clear to somebody in the future in case that decision needs to be revisited.